packages/cli/test/cases/eject.default/eject.default.spec.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Use Case
+ * Run Greenwood eject command to copy core configuration files
+ *
+ * User Result
+ * Should eject configuration files to working directory
+ *
+ * User Command
+ * greenwood eject
+ */
+const fs = require('fs-extra');
+const path = require('path');
+const expect = require('chai').expect;
+const TestBed = require('../../../../../test/test-bed');
+
+describe('Eject Greenwood With: ', function() {
+  let setup;
+
+  before(async function() {
+    setup = new TestBed();
+    this.context = await setup.setupTestBed(__dirname);
+  });
+
+  describe('Default Eject Option', function() {
+
+    before(async function() {
+      await setup.runGreenwoodCommand('eject');
+    });
+
+    it('should output webpack config files to project working directory', function() {
+      let configFiles = fs.readdirSync(__dirname);
+      
+      configFiles = configFiles.filter((file) => file !== 'node_modules' && file !== 'eject.default.spec.js');
+      
+      /* 
+      * 'webpack.config.common.js',
+      * 'webpack.config.develop.js',
+      * 'webpack.config.prod.js'
+      */
+      expect(configFiles.length).to.equal(3);
+    });
+
+    it('should output webpack common config', function() {
+      expect(fs.existsSync(path.join(__dirname, 'webpack.config.common.js'))).to.be.true;
+    });
+
+    it('should output webpack develop config', function() {
+      expect(fs.existsSync(path.join(__dirname, 'webpack.config.develop.js'))).to.be.true;
+    });
+
+    it('should output webpack prod config', function() {
+      expect(fs.existsSync(path.join(__dirname, 'webpack.config.prod.js'))).to.be.true;
+    });
+
+    after(function() {
+      // remove files
+      const configFiles = fs.readdirSync(__dirname);
```

Comment:
should we add nested versions of these files to _.gitignore_ just to they can't ever slip through?

Reply:
I don't know if that's necessary